### DISCUSSION TITLE
Terms of Use and Emails

### DISCUSSION THREAD

**Discussion by kennydude** [etsy/open-api#758] [Q&A]
Hey, I've been looking into developing a solution using the Etsy API and I know at the moment there's no timeframe for providing any realtime/webhook information,

However Etsy provides Emails within minutes of new sales and such, so this would be great for things like stock management by a store configuring these to be parsed automatically

The terms of use for the API doesn't say I can or can't use emails for this information. Could someone from Etsy clarify?

**Comment by nateswart** [comment]
The API doesn't deal with email notifications, so the regular Etsy ToS would probably apply most directly.

**Comment by kbasali** [comment]
@nateswart 
Hello,
Before opening a new thread, I thought I'd ask the question here. The topic is already Etsy ToS.

Can I generate a description with the help of AI for a listing on Etsy?
I didn't see any articles related to AI or ML in Etsy ToS.

Happy christmas

**Reply by Gareth064** [reply]
> @nateswart Hello, Before opening a new thread, I thought I'd ask the question here. The topic is already Etsy ToS.
> 
> Can I generate a description with the help of AI for a listing on Etsy? I didn't see any articles related to AI or ML in Etsy ToS.
> 
> Happy christmas

I'd say that it doesn't matter how you write your descriptions. What would be the difference between getting AI to do it and paying someone on Fiverr to do it. The only thing that would probably rub Etsy up the wrong way is if you straight up copied the descriptions form another sellers listings.

**Reply by kbasali** [reply]
@Gareth064 Thank you for your answer, maybe I couldn't express my concern well.

Let me try to explain again.

Let's say I write my listing descriptions, but they are not creative, attractive, or not well written enough. 
So can I use my listing descriptions as Gtp-3 Prompts and regenerate my descriptions? 

I couldn't find any related articles on this TOS.

https://www.etsy.com/legal/api/
https://www.etsy.com/legal/terms-of-use/

We are planning to add such features to our tools. I'm curious about Etsy @nateswart  and your opinion on the matter.
Gtp-3 and others can be very creative in this area.
